packages/clay-charts/src/GeoMap.tsx

```diff
@@ -255,7 +256,7 @@ const Geomap: React.FunctionComponent<IProps> = ({
 		return () => {
 			forwardRef.current.disposed();
 		};
-	}, []);
+	}, [data, forwardRef, otherProps]);
```

Comment:
Well these are all automated fixes designed to help you follow the "[Rules of Hooks](https://reactjs.org/docs/hooks-rules.html)" — if you don't want to follow the rules, we either have to refactor to avoid triggering them, or suppress them, in which case we'd need a strong rationale for why we think the rules don't apply.